interface-definitions/high-availability.xml.in

```diff
@@ -252,6 +252,144 @@
           </tagNode>
         </children>
       </node>
+      <tagNode name="virtual-server">
+        <properties>
+          <help>Load-balancing virtual server address</help>
+        </properties>
+        <children>
+          <leafNode name="algorithm">
+            <properties>
+              <help>Schedule algorithm (default - least-connection)</help>
+              <completionHelp>
+                <list>round-robin weighted-round-robin least-connection weighted-least-connection source-hashing destination-hashing locality-based-least-connection</list>
+              </completionHelp>
+              <valueHelp>
+                <format>round-robin</format>
+                <description>Round robin</description>
+              </valueHelp>
+              <valueHelp>
+                <format>weighted-round-robin</format>
+                <description>Weighted round robin</description>
+              </valueHelp>
+              <valueHelp>
+                <format>least-connection</format>
+                <description>Least connection</description>
+              </valueHelp>
+              <valueHelp>
+                <format>weighted-least-connection</format>
+                <description>Weighted least connection</description>
+              </valueHelp>
+              <valueHelp>
+                <format>source-hashing</format>
+                <description>Source hashing</description>
+              </valueHelp>
+              <valueHelp>
+                <format>destination-hashing</format>
+                <description>Destination hashing</description>
+              </valueHelp>
+              <valueHelp>
+                <format>locality-based-least-connection</format>
+                <description>Locality-Based least connection</description>
+              </valueHelp>
+              <constraint>
+                <regex>^(round-robin|weighted-round-robin|least-connection|weighted-least-connection|source-hashing|destination-hashing|locality-based-least-connection)$</regex>
+              </constraint>
+            </properties>
+            <defaultValue>least-connection</defaultValue>
+          </leafNode>
+          <leafNode name="delay-loop">
+            <properties>
+              <help>Interval between health-checks (in seconds)</help>
+              <valueHelp>
+                <format>u32:1-600</format>
+                <description>Interval in seconds (default: 10)</description>
+              </valueHelp>
+              <constraint>
+                <validator name="numeric" argument="--range 1-3600"/>
+              </constraint>
+            </properties>
+            <defaultValue>10</defaultValue>
+          </leafNode>
+          <leafNode name="forward-method">
+            <properties>
+              <help>Forwarding method (default - NAT)</help>
```

Comment:
In this file, there are _three_ different formatting variants of the detault value: with a hyphen, with a colon, and without a separator. ;)

Let's standardize on `(default: $default) like on line 305.

Reply:
Done